flask_dance/consumer/oauth2.py

```diff
@@ -243,6 +247,7 @@ def authorized(self):
                 self.token_url,
                 authorization_response=request.url,
                 client_secret=self.client_secret,
+                include_client_id=self.include_client_id,
```

Comment:
Why not just set this in `token_url_params`?

Reply:
Since `oauth2_session.py` lists this as a parameter, I think it would be more clear if this was a standalone parameter in this method as well.

Reply:
Most people using Flask-Dance shouldn't have to think about requests-oauthlib, or whether it's correct to include the client ID, or any of those fiddly OAuth details. The whole point of Flask-Dance is to make it simpler. `token_url_params` provides an opportunity for developers who know those fiddly details to configure them as necessary, but including this option in the top-level arguments (and adding documentation for that option) seems needlessly confusing to me.